bibliopixel/cli/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+
+"""
+
+from .__command__ import *
```

Comment:
I'm very surprised that flake8 doesn't complain here - this is definitely a PEP8 style violation.